gitbook-docs/data_model.md

```diff
@@ -263,6 +263,32 @@ the value should be merged to the full model mounted where the delta‘s context
   ]
 }
 ```
+## Delta Format For Metadata
+
+Metadata can also be specified via a delta. The delta is very similiar to the `values` deltas above, but instead of having `values` key, it will have a `meta` key. Note that a client could multiple meta deltas for any given path from different sources, so the client merge the meta information.
+
+```json
```

Comment:
json examples in documentation must be linked to actual files covered by tests/samples to ensure ongoing validity. See other docs for examples.

Reply:
see PR #491 for change